docs/ban-cooking.rst

```diff
@@ -12,11 +12,16 @@ Usage
 -----
 
 ::
-    ban-cooking <type>
+    ban-cooking <type|all|show>
 
 Valid types are ``booze``, ``brew``, ``fruit``, ``honey``, ``milk``, ``mill``,
 ``oil``, ``seeds`` (i.e. non-tree plants with seeds), ``tallow``, and
-``thread``.
+``thread``. It is possible to include multiple types in a single ban-cooking
+call: ``ban-cooking oil tallow`` will ban both oil and tallow from cooking.
+
+Using ``ban-cooking all`` will ban all types from being cooked.
+
+Using ``ban-cooking show`` will display all types of food that have been banned.
```

Comment:
I'm pretty sure this will have no output. it needs to be specified along other commands that do something.